Wikipedia/Code/AppDelegate.m

```diff
@@ -103,10 +104,28 @@ - (BOOL)application:(UIApplication*)application didFinishLaunchingWithOptions:(N
 
     [self updateDynamicIconShortcutItems];
 
-    NSLog(@"%@", [[[NSFileManager defaultManager] URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask] lastObject]);
+    
+    NSURL *documentsURL = [[[NSFileManager defaultManager] URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask] lastObject];
+    NSLog(@"%@", documentsURL);
 
     self.webServer = [[GCDWebServer alloc] init];
-
+    
+    // HAX: hardcoded for now to match WebView category
+    NSURL *assetsURL = [documentsURL URLByAppendingPathComponent:@"assets"];
+    
+    [self.webServer addDefaultHandlerForMethod:@"GET" requestClass:[GCDWebServerRequest class] processBlock:^GCDWebServerResponse *(GCDWebServerRequest *request) {
```

Comment:
@montehurd I verified that this occurs on the background, but with some new changes this will be using an asyncProcessBlock